src/rust/ide/enso-protocol/src/language_server.rs

```diff
@@ -204,6 +207,150 @@ pub mod response {
         #[allow(missing_docs)]
         pub attributes: FileAttributes,
     }
+
+    /// Response of `open_text_file` method.
+    #[derive(Hash,Debug,Clone,PartialEq,Eq,Serialize,Deserialize)]
+    #[serde(rename_all = "camelCase")]
+    #[allow(missing_docs)]
+    pub struct OpenTextFile {
+        pub write_capability : Option<CapabilityRegistration>,
+        pub content          : String,
+        pub current_version  : Sha3_224
+    }
+
+    /// Response of `create_execution_context` method.
+    #[derive(Hash,Debug,Clone,PartialEq,Eq,Serialize,Deserialize)]
+    #[serde(rename_all = "camelCase")]
+    #[allow(missing_docs)]
+    pub struct CreateExecutionContext {
+        pub can_modify       : CapabilityRegistration,
+        pub receives_updates : CapabilityRegistration
+    }
+}
```

Comment:
It became a bit of file. Perhaps we should move `response` module to a separate file. And I would but all the "types" below also to "types" module in a separate file.